### ISSUE TITLE
False unnecessary semi-colon report

### ISSUE THREAD

**Issue by westonal** [shyiko/ktlint#255]
Version `0.24.0`

When using `should throw the Exception` or `should throw` from kluent you often end up doing this:

```
    @Test
    fun `kluent issue`() {
        val any = Any(); //some setup
        {
            any.toString()
        } `should throw` Exception::class
    }
```

The semi-colon is reported as unnecessary: `Unnecessary semicolon (cannot be auto-corrected)`

But Android Studio doesn't mark it as unnecessary:

![image](https://user-images.githubusercontent.com/6041352/42997539-7d6ca1fc-8bed-11e8-879f-6b2f38f349ab.png)

and removing it causes:

![image](https://user-images.githubusercontent.com/6041352/42997563-937c70c6-8bed-11e8-96ae-493b8e4bdfc1.png)

**Comment by shyiko** [comment]
Fix is coming in 0.25.0.

**Comment by shyiko** [comment]
[0.25.0](https://github.com/shyiko/ktlint/blob/master/CHANGELOG.md#0250---2018-07-25) released.
